### ISSUE TITLE
TypeError: Cannot read property 'toString' of null

### ISSUE THREAD

**Issue by amok** [andzdroid/mongo-express#165]
arises while trying "View" collection

```
TypeError: Cannot read property 'toString' of null
   at Object.exports.to_display (/usr/local/lib/node_modules/mongo-express/filters.js:71:41)
   at eval (eval at <anonymous> (/usr/local/lib/node_modules/mongo-express/node_modules/swig/lib/swig.js:498:13), <anonymous>:327:51)
   at Object.exports.each (/usr/local/lib/node_modules/mongo-express/node_modules/swig/lib/utils.js:45:11)
   at eval (eval at <anonymous> (/usr/local/lib/node_modules/mongo-express/node_modules/swig/lib/swig.js:498:13), <anonymous>:320:10)
   at eval (eval at <anonymous> (/usr/local/lib/node_modules/mongo-express/node_modules/swig/lib/swig.js:498:13), <anonymous>:335:3)
   at Object.exports.each (/usr/local/lib/node_modules/mongo-express/node_modules/swig/lib/utils.js:45:11)
   at eval (eval at <anonymous> (/usr/local/lib/node_modules/mongo-express/node_modules/swig/lib/swig.js:498:13), <anonymous>:306:10)
   at Object.eval [as tpl] (eval at <anonymous> (/usr/local/lib/node_modules/mongo-express/node_modules/swig/lib/swig.js:498:13), <anonymous>:355:3)
   at compiled (/usr/local/lib/node_modules/mongo-express/node_modules/swig/lib/swig.js:619:18)
   at /usr/local/lib/node_modules/mongo-express/node_modules/consolidate/lib/consolidate.js:415:16
   at /usr/local/lib/node_modules/mongo-express/node_modules/consolidate/lib/consolidate.js:146:5
   at tryCatcher (/usr/local/lib/node_modules/mongo-express/node_modules/bluebird/js/main/util.js:26:23)
   at Promise._resolveFromResolver (/usr/local/lib/node_modules/mongo-express/node_modules/bluebird/js/main/promise.js:480:31)
   at new Promise (/usr/local/lib/node_modules/mongo-express/node_modules/bluebird/js/main/promise.js:70:37)
   at promisify (/usr/local/lib/node_modules/mongo-express/node_modules/consolidate/lib/consolidate.js:139:10)
   at Function.exports.swig.render (/usr/local/lib/node_modules/mongo-express/node_modules/consolidate/lib/consolidate.js:408:10)
```

```
➜  npm list --depth=0 -g mongo-express
/usr/local/lib
└── mongo-express@0.29.14
```

**Comment by wulfsolter** [comment]
Fixed with latest (0.29.15)

If issue persists please send link to a gist/pastebin of an export of the collection causing the issue.

**Comment by amok** [comment]
@wulfsolter fixed. thank you
